.github/mergify.yml

```diff
@@ -35,14 +35,14 @@ pull_request_rules:
           - files~=^requirements.*\.txt$
```

Comment:
You may need to change the `check-success` line a few lines up. You can test out the config in the mergify dashboard to see if it works like you expect

Reply:
I added an `e2e-workflow-complete` to match the existing `lint-workflow-complete` and `test-workflow-complete` - will double check the Mergify dashboard as well